src/apps/concepts/api.ts

```diff
@@ -55,7 +55,7 @@ const api = {
           ...optionallyIncludeList("conceptClass", classFilters),
           ...optionallyIncludeList("source", sourceFilters),
           timestamp: new Date().getTime(),
-          includeRetired: includeRetired ? 1 : 0
+          includeRetired
```

Comment:
@jwamalwa  thanks for this, was this passing locally, looking at this (Hide Retired Concepts & Add Filter option to Include Retired Concepts) the statement above the one you introduced looks to be more than a filter

Reply:
@HerbertYiga its a continuation of an already merged work which I explained what I changed.